HACKING.md

```diff
@@ -0,0 +1,31 @@
+# Hacking on Lorax
+
+Here's where to get the code:
+
+    $ git clone https://github.com/weldr/lorax
+    $ cd lorax/
+
+How to build it:
+
+    $ make
+
+## How to run the tests
+
+To run the tests you need the following dependencies installed:
+
+    $ yum install python3-nose python3-pytest-mock python3-pocketlint \
+		python3-mock
+
+Run the basic linting tests like this:
+
+    $ make check
```

Comment:
This doesn't work currently. I've opened https://github.com/weldr/lorax/pull/465 to fix it.